.nrepl-port

```diff
@@ -0,0 +1 @@
+54840
```

Comment:
Is this something that should be added to the repo or possibly added to .gitignore? Disclaimer - I'm not 100% sure what this is for; something involving a REPL I assume.